Uni_life_tracker/app/src/main/java/com/generic/ult/expenses_landing.java

```diff
@@ -0,0 +1,56 @@
+package com.generic.ult;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.util.Patterns;
+import android.view.View;
+import android.widget.Button;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.navigation.ui.AppBarConfiguration;
+import com.google.android.material.textfield.TextInputLayout;
+
+import java.util.Objects;
+
+
+public class expenses_landing extends AppCompatActivity {
```

Comment:
I think the name of a class should be in camelCase

Reply:
Fixed